src/libs/UserUtils.js

```diff
@@ -66,52 +66,52 @@ function getLoginListBrickRoadIndicator(loginList) {
 
 /**
  * Hashes provided string and returns a value between [0, range)
- * @param {String} login
+ * @param {String} text
  * @param {Number} range
  * @returns {Number}
  */
-function hashLogin(login, range) {
-    return Math.abs(hashCode(login.toLowerCase())) % range;
+function hashText(text, range) {
+    return Math.abs(hashCode(text.toLowerCase())) % range;
 }
 
 /**
- * Helper method to return the default avatar associated with the given login
- * @param {String} [login]
+ * Helper method to return the default avatar associated with the given accountID
+ * @param {Number} [accountID]
  * @returns {String}
  */
-function getDefaultAvatar(login = '') {
-    if (!login) {
+function getDefaultAvatar(accountID = -1) {
```

Comment:
Switching to using the `accountID` to get a default avatar caused https://github.com/Expensify/App/issues/28518 as we use an optimistic accountID when chatting with new users.